common/types/head_tracker.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services"
 )
 
+// HeadTracker holds and stores the latest block number experienced by this particular node in a thread safe manner.
```

Comment:
holds and stores latest blocks.

It stores more than just the latest block.

Reply:
thanks for pointing that out prashant. Fixed